### PR TITLE
Testpipeline

### PR DESCRIPTION
**Pull request by denanmusinovic**
pipeline added pull request

### CODE REVIEW

**Comment by denanmusinovic** [comment]
pipeline added

File: .github/workflows/aws.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  unit-tests:
+    name: Unit tests
+    runs-on: ubuntu-latest
+    container: crystallang/crystal
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1      
+      - name: Unit testing
+        run: shards install && crystal spec
+  ameba:
```
Comment (rielas) [inline .github/workflows/aws.yml]: Let's add a newline here to be consistent

```suggestion

    ameba:
```

File: .github/workflows/aws.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    name: Ameba
+    runs-on: ubuntu-latest
+    container:
+      image: neuralegion/devops
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Build binary
+        run: docker build -t builder --target builder .
```
Comment (rielas) [inline .github/workflows/aws.yml]: It's better not to create a Dockerfile just for ameba check. Let's use a [dedicated ameba container](https://hub.docker.com/r/veelenga/ameba/tags).

**Review by rielas — CHANGES_REQUESTED** [review]
1. Let's delete a Dockerfile and use a ameba image
2. Please, rename aws.yml to ci.yml

**Comment by Sija** [comment]
Also, please remove `.ameba.yml` file as it's redundant.

**Review by Sija — CHANGES_REQUESTED** [review]
Please remove `.ameba.yml` file as it's redundant (aside of @rielas' comments).

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+  unit-tests:
+    name: Unit tests
+    runs-on: ubuntu-latest
+    container: crystallang/crystal
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1      
+      - name: Unit testing
+        run: shards install && crystal spec && crystal bin/ameba.cr
```
Comment (rielas) [inline .github/workflows/ci.yml]: It's OK to divide this line on three steps:

1. Shards installing
2. Crystal spec running
3. Ameba

In that case, you can remove ameba job

**Review by rielas — CHANGES_REQUESTED** [review]

**Review by rielas — APPROVED** [review]

**Comment by denanmusinovic** [comment]
I think that's it, project builds properly now.

**Comment by Sija** [comment]
Superseded by #6

**Pull request closed without merging**